terraform/deployments/variables/test/common.tfvars

```diff
@@ -3,5 +3,7 @@
 #--------------------------------------------------------------
 
 # TODO: When it becomes possible, pull these from Terraform "data" type
-mesh_domain = "mesh.govuk-internal.digital"
-mesh_name   = "govuk"
+mesh_subdomain   = "mesh"
+mesh_domain      = "mesh.govuk-internal.digital"
+mesh_name        = "govuk"
+ecs_cluster_name = "ecs"
```

Comment:
What was the intent behind giving the cluster a prefix? Is it so we can define other clusters without having name clashes?